src/core/Akka/Actor/Props.cs

```diff
@@ -105,11 +105,13 @@ private bool CompareDeploy(Props other)
             return Deploy.Equals(other.Deploy);
         }
 
+#pragma warning disable CS0162 // Disabled because it's marked as a TODO
```

Comment:
We should probably fix this issue by closing #599 just comparing if the two supervisor strategies are equal by reference. Too complicated to do it via anything else. What do you think @sean-gilliam ?

Reply:
Also, we have a serializable supervisor strategy but no one uses it because the syntax is extremely awkward.